python/tvm/relay/transform/transform.py

```diff
@@ -1099,6 +1099,19 @@ def SimplifyExpr():
     return _ffi_api.SimplifyExpr()
 
 
+def ImplicitPadding():
```

Comment:
If we build up multiple patterns that can fuse implicit padding it may be useful to key on/off certain patterns based on operator type via a white or blacklist.